pkg/util/namespace.go

```diff
@@ -10,6 +10,11 @@ import (
 	kubeclient "k8s.io/client-go/kubernetes"
 )
 
+const (
+	namespaceAnnotationKey   = "karmada.io/namespace"
+	namespaceAnnotationValue = "created-by-karmada"
```

Comment:
I might be ambiguous, do you think the namespace named `karmada-system` which is created by `karmadactl init` belongs to `created-by-karmada`? I think yes.